### DISCUSSION TITLE
Provide example for development to handle tag literals

### DISCUSSION THREAD

**Discussion by gaberger** [mikub/titanoboa#28] [General]
Need to expose data-readers for the following in development

#titanoboa.exp/Expression{:value "(fn [p] \n true)"
                                                              :type "clojure"}}

**Comment by mikub** [comment]
The readers etc. are in 
https://github.com/mikub/titanoboa/blob/master/src/clj/titanoboa/exp.clj

Btw. depending on use case you might not even need to always use titanoboa.exp.Expression - see https://github.com/mikub/titanoboa/wiki/Designing-Workflows#workload-fn
